R/01-createNewDBSource.R

```diff
@@ -0,0 +1,168 @@
+#' Create New DB Source
+#'
+#' Creates a script for a new data source from a database connection and sets .Renviron variables.
+#' Only "mySql" databases are supported.
+#'
+#' @param dbName (character) name of the new database source, e.g. "xyDBname"
+#' @param tableName name of the table containing the data
+#' @param descriptionCreator (character) command that creates the description, e.g. pasting data
+#'  columns "var1" and "var2": "paste(isoData$var1, isoData$var2)"
+#' @param datingType (character) dating type for the database
+#' @param coordType (character) coord type for the database
```

Comment:
I don't understand those two arguments. Maybe this is obvious with more domain knowledge. Maybe an example would help. Like what are the datingTypes / coordTypes for existing data sources.

Reply:
I updated the description with some examples. But I found nothing in this package or the app package regarding these parameters.